### PR TITLE
Guides: Mention rake routes task

### PR DESCRIPTION
**Pull request by dacook**
[skip ci]

### Summary
The Routing guide doesn't mention the `rake routes` task, which is a very helpful tool in understanding route helper names. These can sometimes get quite long, especially including namespaces. So it would be really helpful for a developer to learn how to discover these.

I initially wanted to write a formula for guessing helper names (eg "it is a concatenation of action name, namespace(s), controller name"), but then realised this tip is more helpful as it is more direct.

### Other Information
N/A

### CODE REVIEW

**Comment by ceritium** [comment]
Hi @dacook, there is a section dedicated to [listing existing routes](https://guides.rubyonrails.org/routing.html#listing-existing-routes).

**Review by ceritium — COMMENTED** [review]
Hi @dacook, there is a section dedicated to [listing existing routes](https://guides.rubyonrails.org/routing.html#listing-existing-routes). It already mentions `bin/rails routes`.

**Comment by dacook** [comment]
How embarrassing, I guess I was searching for 'rake' and didn't find it! Is
it worth linking to that section here?

File: guides/source/routing.md [hunk 1]
```diff
@@ -142,6 +142,8 @@ Creating a resourceful route will also expose a number of helpers to the control
 
 Each of these helpers has a corresponding `_url` helper (such as `photos_url`) which returns the same path prefixed with the current host, port, and path prefix.
 
+TIP: You can generate a full list of paths and helper names with `rake routes`.
```
Comment (eileencodes) [inline guides/source/routing.md:145]: Docs elsewhere are using `rails routes`, can you use that here too?

Comment (dacook) [reply guides/source/routing.md:145]: Yes you're right, I am out of date. I'm now aware this is mentioned elsewhere on the page, so will update my PR

**Pull request merged**